odl/phantom/geometric.py

```diff
@@ -95,6 +95,92 @@ def phantom(x):
     return space.element(phantom)
 
 
+def defrise(space, nellipses=8, alternating=False):
+    """Phantom with regularily spaced ellipses.
```

Comment:
Ah, now that I see this: One feedback I got was the term "ellipses" that doesn't make sense in 3D. We should call things "ellipsoid" throughout. But let's make a new issue for that. (Read: I'll create a new issue.)